keps/sig-storage/1790-recover-resize-failure/README.md

```diff
@@ -360,7 +393,7 @@ _This section must be completed when targeting beta graduation to a release._
 
 ## Drawbacks
 
-- One drawback is while this KEP allows an user to reduce requested PVC size, it does not automatically reduce quota.
+- One drawback is while this KEP allows an user to reduce requested PVC size, it does not allow reduction in size all the way to same value as original size recorded in `pvc.Status.Cap`. The reason is currently resize controller and kubelet work on a volume one after the other and we need special logic in both kubelet and control-plane to process reduction in size all the way upto original value. This is somewhat racy and we need a signaling mechanism between control-plane and kubelet when `pvc.Status.AllocatedResources` needs to be adjusted. We plan to revisit and address this in next release.
```

Comment:
At a first glance I think we should be able to use the new ResizeStatus to do something like `orig size == new size && resizeStatus is pending/inprogress/failed`. Agree we revisit before moving to beta.